File: server/src/main/java/net/bytemc/minestom/server/ByteServerBootstrap.java
package net.bytemc.minestom.server;

import net.minestom.server.MinecraftServer;

public class ByteServerBootstrap {

    public static void main(String[] args) {
        System.out.println("[ByteServer] starting minestom server...");

        var server = MinecraftServer.init();

        System.out.println("[ByteServer] initializing byteServer...");
        new ByteServer(server, args);
    }
}

File: server/src/main/java/net/bytemc/minestom/server/ByteServerBootstrap.kt
package net.bytemc.minestom.server

import net.minestom.server.MinecraftServer

fun main(args: Array<String>) {
    println("[ByteServer] initializing minestom server...")

    val server = MinecraftServer.init()

    println("[ByteServer] initializing byteServer...")
    ByteServer(server, args)
}